ksqldb-rest-app/src/test/resources/ksql-plan-schema/schema.json

```diff
@@ -427,6 +429,28 @@
       "title" : "streamSelectKeyV1",
       "required" : [ "@type", "properties", "source", "keyExpression" ]
     },
+    "StreamSelectKey" : {
+      "type" : "object",
+      "additionalProperties" : false,
+      "properties" : {
+        "@type" : {
+          "type" : "string",
+          "enum" : [ "streamSelectKeyV2" ],
+          "default" : "streamSelectKeyV2"
```

Comment:
will we be able to change this in the future or will we always have a `V2` stuck in there? that seems like it might be motivation to go down the boolean path...

Reply:
The whole point of Rohan's command topic work was to allow different versions of steps!  The pre-existing version is called `StreamSelectKeyV1`: all the step names have the version appended....